### ISSUE TITLE
grounded DINO-V2

### ISSUE THREAD

**Issue by kobic8** [autodistill/autodistill#16]
Hello,
are you planning on integrating DINO-2 for faster and more accurate results for bbox+labels, rather than grounded-dino?

**Comment by capjamesg** [comment]
Hello! Are you talking about [DINOv2 by Meta Research](https://dinov2.metademolab.com/)? If not, can you share a link to the research paper! We're keen to add new state-of-the-art models as extensions to the `autodistill` ecosystem.

**Comment by kobic8** [comment]
Hello!!:) yes, I was talking about DINO-2 by Meta. B.t.w. this is very confusing since groundingDINO is based on a differet model named DINO as well, by IDEA-Reasearch

**Comment by capjamesg** [comment]
Thank you for the clarification! The DINOv2 paper does not mention object detection as an application, so we will not be implementing it at this stage. We are working on DINOv2 support, but for classification with SVM (DINOv2 for classification returns embeddings, so we need to add an additional classifier on top that can work with those embeddings). As new foundation object detection models are released, we will look at adding them to `autodistill`!

**Comment by kobic8** [comment]
Thanks so much for the reply James, and I am following you, and this repo for more updates. Although there is something that is not quite clear: Knowledge distillation is used to compress a complex and large neural network into a smaller and simpler one, while still retaining the accuracy and performance of the resultant model. This process involves training a smaller neural network to mimic the behavior of a larger and more complex "teacher" network by learning from its predictions or internal representations.

With `autodistill!` we can auto-annotate through a base model e.g., G-DINO and train with YOLO8 for example. My issue is: what if the base model doesn't recognize the labels (name of specific boats I have in a marine dataset), how can I fine-tune it? so the model will find a boat, but will also "know" the brand of the boat for example?

**Comment by yeldarby** [comment]
> My issue is: what if the base model doesn't recognize the labels (name of specific boats I have in a marine dataset), how can I fine-tune it? so the model will find a boat, but will also "know" the brand of the boat for example?

This is still an open question in research! The short answer is there are still many problems you can't do without a human in the loop yet.

**Comment by SkalskiP** [comment]
@kobic8 I doubt it will ever know the specific brand of your boat, but you can try prompt engineering. For example, look for ships in specific colors or shapes.
